Update to Quarkus `v2.15.0` fails due to the missing class `graphql.parser.antlr.GraphqlLexer`

See here https://github.com/stargate/stargate/actions/runs/3705060664/jobs/6278555444, the unit tests for GraphQL fail with:

```
java.lang.NoClassDefFoundError: Could not initialize class graphql.parser.antlr.GraphqlLexer
```

I am not really sure what's going on, aren't we using our own GraphQL version and how can the Quarkus update be responsible for this? In the release notes the GraphQL was mentioned for supporting federation now: https://quarkus.io/blog/quarkus-2-15-0-final-released/

@kathirsvn Mind if you have a look?

Seems the error can easily be received by running the `io.stargate.sgv2.graphql.schema.graphqlfirst.processor.QueryMappingModelTest`: 

```
/usr/lib/jvm/java-1.17.0-openjdk-amd64/bin/java -ea -Djava.util.logging.manager=org.jboss.logmanager.LogManager -Xmx4g -Didea.test.cyclic.buffer.size=1048576 -javaagent:/snap/intellij-idea-community/402/lib/idea_rt.jar=43505:/snap/intellij-idea-community/402/bin -Dfile.encoding=UTF-8 -classpath /home/ise/.m2/repository/org/junit/vintage/junit-vintage-engine/5.9.1/junit-vintage-engine-5.9.1.jar:/snap/intellij-idea-community/402/lib/idea_rt.jar:/snap/intellij-idea-community/402/plugins/junit/lib/junit5-rt.jar:/snap/intellij-idea-community/402/plugins/junit/lib/junit-rt.jar:/home/ise/workspace/datastax/stargate/stargate/apis/sgv2-graphqlapi/target/test-classes:/home/ise/workspace/datastax/stargate/stargate/apis/sgv2-graphqlapi/target/classes:/home/ise/workspace/datastax/stargate/stargate/apis/sgv2-quarkus-common/target/classes:/home/ise/.m2/repository/io/stargate/grpc/bridge-proto/2.0.4-SNAPSHOT/bridge-proto-2.0.4-SNAPSHOT.jar:/home/ise/.m2/repository/io/grpc/grpc-protobuf/1.51.0/grpc-protobuf-1.51.0.jar:/home/ise/.m2/repository/io/grpc/grpc-api/1.51.0/grpc-api-1.51.0.jar:/home/ise/.m2/repository/io/grpc/grpc-context/1.51.0/grpc-context-1.51.0.jar:/home/ise/.m2/repository/com/google/api/grpc/proto-google-common-protos/2.10.0/proto-google-common-protos-2.10.0.jar:/home/ise/.m2/repository/io/grpc/grpc-protobuf-lite/1.51.0/grpc-protobuf-lite-1.51.0.jar:/home/ise/.m2/repository/com/google/guava/guava/31.1-jre/guava-31.1-jre.jar:/home/ise/.m2/repository/com/google/guava/failureaccess/1.0.1/failureaccess-1.0.1.jar:/home/ise/.m2/repository/org/checkerframework/checker-qual/3.27.0/checker-qual-3.27.0.jar:/home/ise/.m2/repository/com/google/j2objc/j2objc-annotations/1.3/j2objc-annotations-1.3.jar:/home/ise/.m2/repository/io/grpc/grpc-stub/1.51.0/grpc-stub-1.51.0.jar:/home/ise/.m2/repository/com/google/errorprone/error_prone_annotations/2.16/error_prone_annotations-2.16.jar:/home/ise/.m2/repository/io/quarkus/quarkus-config-yaml/2.15.0.Final/quarkus-config-yaml-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/config/smallrye-config-source-yaml/2.13.0/smallrye-config-source-yaml-2.13.0.jar:/home/ise/.m2/repository/org/yaml/snakeyaml/1.33/snakeyaml-1.33.jar:/home/ise/.m2/repository/io/smallrye/config/smallrye-config-common/2.13.0/smallrye-config-common-2.13.0.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-classloader/1.13.2/smallrye-common-classloader-1.13.2.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-constraint/1.13.2/smallrye-common-constraint-1.13.2.jar:/home/ise/.m2/repository/org/eclipse/microprofile/config/microprofile-config-api/2.0.1/microprofile-config-api-2.0.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-openapi/2.15.0.Final/quarkus-smallrye-openapi-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/smallrye-open-api-core/2.3.1/smallrye-open-api-core-2.3.1.jar:/home/ise/.m2/repository/org/eclipse/microprofile/openapi/microprofile-openapi-api/2.0.1/microprofile-openapi-api-2.0.1.jar:/home/ise/.m2/repository/com/fasterxml/jackson/dataformat/jackson-dataformat-yaml/2.14.1/jackson-dataformat-yaml-2.14.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-swagger-ui/2.15.0.Final/quarkus-swagger-ui-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-grpc/2.15.0.Final/quarkus-grpc-2.15.0.Final.jar:/home/ise/.m2/repository/jakarta/annotation/jakarta.annotation-api/1.3.5/jakarta.annotation-api-1.3.5.jar:/home/ise/.m2/repository/io/quarkus/quarkus-grpc-api/2.15.0.Final/quarkus-grpc-api-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-grpc-common/2.15.0.Final/quarkus-grpc-common-2.15.0.Final.jar:/home/ise/.m2/repository/io/vertx/vertx-grpc/4.3.5/vertx-grpc-4.3.5.jar:/home/ise/.m2/repository/io/grpc/grpc-netty/1.51.0/grpc-netty-1.51.0.jar:/home/ise/.m2/repository/io/grpc/grpc-core/1.51.0/grpc-core-1.51.0.jar:/home/ise/.m2/repository/com/google/code/gson/gson/2.10/gson-2.10.jar:/home/ise/.m2/repository/io/perfmark/perfmark-api/0.25.0/perfmark-api-0.25.0.jar:/home/ise/.m2/repository/io/vertx/vertx-grpc-server/4.3.5/vertx-grpc-server-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-grpc-common/4.3.5/vertx-grpc-common-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-grpc-client/4.3.5/vertx-grpc-client-4.3.5.jar:/home/ise/.m2/repository/io/quarkus/security/quarkus-security/1.1.4.Final/quarkus-security-1.1.4.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-stork/2.15.0.Final/quarkus-smallrye-stork-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/stork/stork-api/1.3.3/stork-api-1.3.3.jar:/home/ise/.m2/repository/io/smallrye/stork/stork-core/1.3.3/stork-core-1.3.3.jar:/home/ise/.m2/repository/io/quarkus/quarkus-grpc-stubs/2.15.0.Final/quarkus-grpc-stubs-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-annotation/1.13.2/smallrye-common-annotation-1.13.2.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-vertx-context/1.13.2/smallrye-common-vertx-context-1.13.2.jar:/home/ise/.m2/repository/io/vertx/vertx-core/4.3.5/vertx-core-4.3.5.jar:/home/ise/.m2/repository/io/netty/netty-handler-proxy/4.1.85.Final/netty-handler-proxy-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-codec-socks/4.1.85.Final/netty-codec-socks-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-codec-http/4.1.85.Final/netty-codec-http-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-codec-http2/4.1.85.Final/netty-codec-http2-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-resolver-dns/4.1.85.Final/netty-resolver-dns-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-codec-dns/4.1.85.Final/netty-codec-dns-4.1.85.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-hibernate-validator/2.15.0.Final/quarkus-hibernate-validator-2.15.0.Final.jar:/home/ise/.m2/repository/org/hibernate/validator/hibernate-validator/6.2.5.Final/hibernate-validator-6.2.5.Final.jar:/home/ise/.m2/repository/jakarta/validation/jakarta.validation-api/2.0.2/jakarta.validation-api-2.0.2.jar:/home/ise/.m2/repository/com/fasterxml/classmate/1.5.1/classmate-1.5.1.jar:/home/ise/.m2/repository/org/glassfish/jakarta.el/3.0.4/jakarta.el-3.0.4.jar:/home/ise/.m2/repository/io/smallrye/config/smallrye-config-validator/2.13.0/smallrye-config-validator-2.13.0.jar:/home/ise/.m2/repository/org/jboss/spec/javax/ws/rs/jboss-jaxrs-api_2.1_spec/2.0.1.Final/jboss-jaxrs-api_2.1_spec-2.0.1.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-resteasy-reactive/2.15.0.Final/quarkus-resteasy-reactive-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-resteasy-reactive-common/2.15.0.Final/quarkus-resteasy-reactive-common-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/resteasy/reactive/resteasy-reactive-common/2.15.0.Final/resteasy-reactive-common-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/resteasy/reactive/resteasy-reactive-common-types/2.15.0.Final/resteasy-reactive-common-types-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/resteasy/reactive/resteasy-reactive-vertx/2.15.0.Final/resteasy-reactive-vertx-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-core/2.28.0/smallrye-mutiny-vertx-core-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-runtime/2.28.0/smallrye-mutiny-vertx-runtime-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/vertx-mutiny-generator/2.28.0/vertx-mutiny-generator-2.28.0.jar:/home/ise/.m2/repository/io/vertx/vertx-codegen/4.3.5/vertx-codegen-4.3.5.jar:/home/ise/.m2/repository/io/quarkus/resteasy/reactive/resteasy-reactive/2.15.0.Final/resteasy-reactive-2.15.0.Final.jar:/home/ise/.m2/repository/org/jboss/spec/javax/xml/bind/jboss-jaxb-api_2.3_spec/2.0.0.Final/jboss-jaxb-api_2.3_spec-2.0.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-resteasy-reactive-jackson/2.15.0.Final/quarkus-resteasy-reactive-jackson-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-resteasy-reactive-jackson-common/2.15.0.Final/quarkus-resteasy-reactive-jackson-common-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/resteasy/reactive/resteasy-reactive-jackson/2.15.0.Final/resteasy-reactive-jackson-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-jackson/2.15.0.Final/quarkus-jackson-2.15.0.Final.jar:/home/ise/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jsr310/2.14.1/jackson-datatype-jsr310-2.14.1.jar:/home/ise/.m2/repository/com/fasterxml/jackson/datatype/jackson-datatype-jdk8/2.14.1/jackson-datatype-jdk8-2.14.1.jar:/home/ise/.m2/repository/com/fasterxml/jackson/module/jackson-module-parameter-names/2.14.1/jackson-module-parameter-names-2.14.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-fault-tolerance/2.15.0.Final/quarkus-smallrye-fault-tolerance-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance/5.6.0/smallrye-fault-tolerance-5.6.0.jar:/home/ise/.m2/repository/org/eclipse/microprofile/fault-tolerance/microprofile-fault-tolerance-api/3.0/microprofile-fault-tolerance-api-3.0.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-api/5.6.0/smallrye-fault-tolerance-api-5.6.0.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-core/5.6.0/smallrye-fault-tolerance-core-5.6.0.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-autoconfig-core/5.6.0/smallrye-fault-tolerance-autoconfig-core-5.6.0.jar:/home/ise/.m2/repository/jakarta/enterprise/jakarta.enterprise.cdi-api/2.0.2/jakarta.enterprise.cdi-api-2.0.2.jar:/home/ise/.m2/repository/jakarta/el/jakarta.el-api/3.0.3/jakarta.el-api-3.0.3.jar:/home/ise/.m2/repository/org/jboss/logging/commons-logging-jboss-logging/1.0.0.Final/commons-logging-jboss-logging-1.0.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-context-propagation/2.15.0.Final/quarkus-smallrye-context-propagation-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/smallrye-context-propagation/1.2.2/smallrye-context-propagation-1.2.2.jar:/home/ise/.m2/repository/io/smallrye/smallrye-context-propagation-api/1.2.2/smallrye-context-propagation-api-1.2.2.jar:/home/ise/.m2/repository/io/smallrye/smallrye-context-propagation-storage/1.2.2/smallrye-context-propagation-storage-1.2.2.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-context-propagation/5.6.0/smallrye-fault-tolerance-context-propagation-5.6.0.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-mutiny/5.6.0/smallrye-fault-tolerance-mutiny-5.6.0.jar:/home/ise/.m2/repository/io/quarkus/quarkus-security/2.15.0.Final/quarkus-security-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-security-runtime-spi/2.15.0.Final/quarkus-security-runtime-spi-2.15.0.Final.jar:/home/ise/.m2/repository/jakarta/interceptor/jakarta.interceptor-api/1.2.5/jakarta.interceptor-api-1.2.5.jar:/home/ise/.m2/repository/io/quarkus/quarkus-cache/2.15.0.Final/quarkus-cache-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-caffeine/2.15.0.Final/quarkus-caffeine-2.15.0.Final.jar:/home/ise/.m2/repository/com/github/ben-manes/caffeine/caffeine/3.1.1/caffeine-3.1.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-micrometer-registry-prometheus/2.15.0.Final/quarkus-micrometer-registry-prometheus-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-micrometer/2.15.0.Final/quarkus-micrometer-2.15.0.Final.jar:/home/ise/.m2/repository/io/micrometer/micrometer-core/1.10.2/micrometer-core-1.10.2.jar:/home/ise/.m2/repository/io/micrometer/micrometer-commons/1.10.2/micrometer-commons-1.10.2.jar:/home/ise/.m2/repository/io/micrometer/micrometer-observation/1.10.2/micrometer-observation-1.10.2.jar:/home/ise/.m2/repository/org/latencyutils/LatencyUtils/2.0.3/LatencyUtils-2.0.3.jar:/home/ise/.m2/repository/io/micrometer/micrometer-registry-prometheus/1.10.2/micrometer-registry-prometheus-1.10.2.jar:/home/ise/.m2/repository/io/prometheus/simpleclient_common/0.16.0/simpleclient_common-0.16.0.jar:/home/ise/.m2/repository/io/prometheus/simpleclient/0.16.0/simpleclient-0.16.0.jar:/home/ise/.m2/repository/io/prometheus/simpleclient_tracer_otel/0.16.0/simpleclient_tracer_otel-0.16.0.jar:/home/ise/.m2/repository/io/prometheus/simpleclient_tracer_common/0.16.0/simpleclient_tracer_common-0.16.0.jar:/home/ise/.m2/repository/io/prometheus/simpleclient_tracer_otel_agent/0.16.0/simpleclient_tracer_otel_agent-0.16.0.jar:/home/ise/.m2/repository/io/quarkus/quarkus-opentelemetry/2.15.0.Final/quarkus-opentelemetry-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-vertx/2.15.0.Final/quarkus-vertx-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-netty/2.15.0.Final/quarkus-netty-2.15.0.Final.jar:/home/ise/.m2/repository/com/aayushatharva/brotli4j/brotli4j/1.8.0/brotli4j-1.8.0.jar:/home/ise/.m2/repository/com/aayushatharva/brotli4j/native-linux-x86_64/1.8.0/native-linux-x86_64-1.8.0.jar:/home/ise/.m2/repository/io/netty/netty-codec-haproxy/4.1.85.Final/netty-codec-haproxy-4.1.85.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-vertx-latebound-mdc-provider/2.15.0.Final/quarkus-vertx-latebound-mdc-provider-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/smallrye-fault-tolerance-vertx/5.6.0/smallrye-fault-tolerance-vertx-5.6.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk/1.19.0/opentelemetry-sdk-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-api/1.19.0/opentelemetry-api-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-context/1.19.0/opentelemetry-context-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk-common/1.19.0/opentelemetry-sdk-common-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk-trace/1.19.0/opentelemetry-sdk-trace-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk-metrics/1.19.0/opentelemetry-sdk-metrics-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk-logs/1.19.0-alpha/opentelemetry-sdk-logs-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-api-logs/1.19.0-alpha/opentelemetry-api-logs-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-extension-annotations/1.18.0/opentelemetry-extension-annotations-1.18.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-sdk-extension-autoconfigure-spi/1.19.0/opentelemetry-sdk-extension-autoconfigure-spi-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-semconv/1.19.0-alpha/opentelemetry-semconv-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/instrumentation/opentelemetry-instrumentation-api/1.19.0/opentelemetry-instrumentation-api-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/instrumentation/opentelemetry-instrumentation-annotations/1.19.0-alpha/opentelemetry-instrumentation-annotations-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/instrumentation/opentelemetry-instrumentation-annotations-support/1.19.0-alpha/opentelemetry-instrumentation-annotations-support-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/instrumentation/opentelemetry-instrumentation-api-semconv/1.19.0-alpha/opentelemetry-instrumentation-api-semconv-1.19.0-alpha.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-exporter-otlp-common/1.19.0/opentelemetry-exporter-otlp-common-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-exporter-common/1.19.0/opentelemetry-exporter-common-1.19.0.jar:/home/ise/.m2/repository/io/opentelemetry/opentelemetry-exporter-otlp/1.19.0/opentelemetry-exporter-otlp-1.19.0.jar:/home/ise/.m2/repository/commons-codec/commons-codec/1.15/commons-codec-1.15.jar:/home/ise/.m2/repository/io/quarkus/quarkus-arc/2.15.0.Final/quarkus-arc-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/arc/arc/2.15.0.Final/arc-2.15.0.Final.jar:/home/ise/.m2/repository/jakarta/transaction/jakarta.transaction-api/1.3.3/jakarta.transaction-api-1.3.3.jar:/home/ise/.m2/repository/io/smallrye/reactive/mutiny/1.8.0/mutiny-1.8.0.jar:/home/ise/.m2/repository/org/jboss/logging/jboss-logging/3.5.0.Final/jboss-logging-3.5.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-core/2.15.0.Final/quarkus-core-2.15.0.Final.jar:/home/ise/.m2/repository/jakarta/inject/jakarta.inject-api/1.0/jakarta.inject-api-1.0.jar:/home/ise/.m2/repository/io/quarkus/quarkus-ide-launcher/2.15.0.Final/quarkus-ide-launcher-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-development-mode-spi/2.15.0.Final/quarkus-development-mode-spi-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/config/smallrye-config/2.13.0/smallrye-config-2.13.0.jar:/home/ise/.m2/repository/io/smallrye/config/smallrye-config-core/2.13.0/smallrye-config-core-2.13.0.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-expression/1.13.2/smallrye-common-expression-1.13.2.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-function/1.13.2/smallrye-common-function-1.13.2.jar:/home/ise/.m2/repository/org/jboss/logmanager/jboss-logmanager-embedded/1.0.11/jboss-logmanager-embedded-1.0.11.jar:/home/ise/.m2/repository/org/jboss/logging/jboss-logging-annotations/2.2.1.Final/jboss-logging-annotations-2.2.1.Final.jar:/home/ise/.m2/repository/org/jboss/threads/jboss-threads/3.4.3.Final/jboss-threads-3.4.3.Final.jar:/home/ise/.m2/repository/org/jboss/slf4j/slf4j-jboss-logmanager/1.2.0.Final/slf4j-jboss-logmanager-1.2.0.Final.jar:/home/ise/.m2/repository/org/graalvm/sdk/graal-sdk/22.3.0/graal-sdk-22.3.0.jar:/home/ise/.m2/repository/org/wildfly/common/wildfly-common/1.5.4.Final-format-001/wildfly-common-1.5.4.Final-format-001.jar:/home/ise/.m2/repository/io/quarkus/quarkus-bootstrap-runner/2.15.0.Final/quarkus-bootstrap-runner-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-fs-util/0.0.9/quarkus-fs-util-0.0.9.jar:/home/ise/.m2/repository/org/eclipse/microprofile/context-propagation/microprofile-context-propagation-api/1.2/microprofile-context-propagation-api-1.2.jar:/home/ise/.m2/repository/io/quarkus/quarkus-container-image-docker/2.15.0.Final/quarkus-container-image-docker-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-container-image/2.15.0.Final/quarkus-container-image-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-health/2.15.0.Final/quarkus-smallrye-health-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/smallrye-health/3.3.1/smallrye-health-3.3.1.jar:/home/ise/.m2/repository/org/eclipse/microprofile/health/microprofile-health-api/3.1/microprofile-health-api-3.1.jar:/home/ise/.m2/repository/io/smallrye/smallrye-health-api/3.3.1/smallrye-health-api-3.3.1.jar:/home/ise/.m2/repository/io/smallrye/smallrye-health-provided-checks/3.3.1/smallrye-health-provided-checks-3.3.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-vertx-http/2.15.0.Final/quarkus-vertx-http-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-credentials/2.15.0.Final/quarkus-credentials-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-vertx-http-dev-console-runtime-spi/2.15.0.Final/quarkus-vertx-http-dev-console-runtime-spi-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-web/2.28.0/smallrye-mutiny-vertx-web-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-web-common/2.28.0/smallrye-mutiny-vertx-web-common-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-auth-common/2.28.0/smallrye-mutiny-vertx-auth-common-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-bridge-common/2.28.0/smallrye-mutiny-vertx-bridge-common-2.28.0.jar:/home/ise/.m2/repository/io/smallrye/reactive/smallrye-mutiny-vertx-uri-template/2.28.0/smallrye-mutiny-vertx-uri-template-2.28.0.jar:/home/ise/.m2/repository/io/vertx/vertx-uri-template/4.3.5/vertx-uri-template-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-web/4.3.5/vertx-web-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-web-common/4.3.5/vertx-web-common-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-auth-common/4.3.5/vertx-auth-common-4.3.5.jar:/home/ise/.m2/repository/io/vertx/vertx-bridge-common/4.3.5/vertx-bridge-common-4.3.5.jar:/home/ise/.m2/repository/io/github/crac/org-crac/0.1.3/org-crac-0.1.3.jar:/home/ise/.m2/repository/io/quarkus/quarkus-jsonp/2.15.0.Final/quarkus-jsonp-2.15.0.Final.jar:/home/ise/.m2/repository/org/glassfish/jakarta.json/1.1.6/jakarta.json-1.1.6.jar:/home/ise/.m2/repository/com/graphql-java/graphql-java/18.3/graphql-java-18.3.jar:/home/ise/.m2/repository/com/graphql-java/java-dataloader/3.1.2/java-dataloader-3.1.2.jar:/home/ise/.m2/repository/org/reactivestreams/reactive-streams/1.0.3/reactive-streams-1.0.3.jar:/home/ise/.m2/repository/org/antlr/antlr4-runtime/4.10.1/antlr4-runtime-4.10.1.jar:/home/ise/.m2/repository/org/slf4j/slf4j-api/1.7.36/slf4j-api-1.7.36.jar:/home/ise/.m2/repository/com/apollographql/federation/federation-graphql-java-support/2.0.3/federation-graphql-java-support-2.0.3.jar:/home/ise/.m2/repository/com/apollographql/federation/federation-graphql-java-support-api/2.0.3/federation-graphql-java-support-api-2.0.3.jar:/home/ise/.m2/repository/com/google/protobuf/protobuf-java/3.21.9/protobuf-java-3.21.9.jar:/home/ise/workspace/datastax/stargate/stargate/apis/sgv2-quarkus-common/target/test-classes:/home/ise/.m2/repository/org/testcontainers/junit-jupiter/1.17.6/junit-jupiter-1.17.6.jar:/home/ise/.m2/repository/org/testcontainers/testcontainers/1.17.6/testcontainers-1.17.6.jar:/home/ise/.m2/repository/junit/junit/4.13.2/junit-4.13.2.jar:/home/ise/.m2/repository/org/apache/commons/commons-compress/1.22/commons-compress-1.22.jar:/home/ise/.m2/repository/org/rnorth/duct-tape/duct-tape/1.0.8/duct-tape-1.0.8.jar:/home/ise/.m2/repository/com/github/docker-java/docker-java-api/3.2.13/docker-java-api-3.2.13.jar:/home/ise/.m2/repository/com/github/docker-java/docker-java-transport-zerodep/3.2.13/docker-java-transport-zerodep-3.2.13.jar:/home/ise/.m2/repository/com/github/docker-java/docker-java-transport/3.2.13/docker-java-transport-3.2.13.jar:/home/ise/.m2/repository/net/java/dev/jna/jna/5.8.0/jna-5.8.0.jar:/home/ise/.m2/repository/org/junit/jupiter/junit-jupiter-api/5.9.1/junit-jupiter-api-5.9.1.jar:/home/ise/.m2/repository/org/opentest4j/opentest4j/1.2.0/opentest4j-1.2.0.jar:/home/ise/.m2/repository/org/junit/platform/junit-platform-commons/1.9.1/junit-platform-commons-1.9.1.jar:/home/ise/.m2/repository/org/apiguardian/apiguardian-api/1.1.2/apiguardian-api-1.1.2.jar:/home/ise/.m2/repository/io/quarkus/quarkus-junit5/2.15.0.Final/quarkus-junit5-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-bootstrap-core/2.15.0.Final/quarkus-bootstrap-core-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-bootstrap-app-model/2.15.0.Final/quarkus-bootstrap-app-model-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/common/smallrye-common-io/1.13.2/smallrye-common-io-1.13.2.jar:/home/ise/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.inject/0.3.5/org.eclipse.sisu.inject-0.3.5.jar:/home/ise/.m2/repository/io/quarkus/quarkus-test-common/2.15.0.Final/quarkus-test-common-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-core-deployment/2.15.0.Final/quarkus-core-deployment-2.15.0.Final.jar:/home/ise/.m2/repository/org/aesh/readline/2.2/readline-2.2.jar:/home/ise/.m2/repository/org/fusesource/jansi/jansi/1.18/jansi-1.18.jar:/home/ise/.m2/repository/org/aesh/aesh/2.6/aesh-2.6.jar:/home/ise/.m2/repository/io/quarkus/gizmo/gizmo/1.4.0.Final/gizmo-1.4.0.Final.jar:/home/ise/.m2/repository/org/ow2/asm/asm/9.4/asm-9.4.jar:/home/ise/.m2/repository/org/ow2/asm/asm-commons/9.4/asm-commons-9.4.jar:/home/ise/.m2/repository/io/quarkus/quarkus-class-change-agent/2.15.0.Final/quarkus-class-change-agent-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-devtools-utilities/2.15.0.Final/quarkus-devtools-utilities-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-builder/2.15.0.Final/quarkus-builder-2.15.0.Final.jar:/home/ise/.m2/repository/org/junit/platform/junit-platform-launcher/1.9.1/junit-platform-launcher-1.9.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-bootstrap-maven-resolver/2.15.0.Final/quarkus-bootstrap-maven-resolver-2.15.0.Final.jar:/home/ise/.m2/repository/org/apache/maven/maven-embedder/3.8.6/maven-embedder-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-settings/3.8.6/maven-settings-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-core/3.8.6/maven-core-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-artifact/3.8.6/maven-artifact-3.8.6.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-component-annotations/2.1.0/plexus-component-annotations-2.1.0.jar:/home/ise/.m2/repository/org/apache/maven/maven-plugin-api/3.8.6/maven-plugin-api-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-model/3.8.6/maven-model-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-model-builder/3.8.6/maven-model-builder-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-builder-support/3.8.6/maven-builder-support-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-api/1.6.3/maven-resolver-api-1.6.3.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-util/1.6.3/maven-resolver-util-1.6.3.jar:/home/ise/.m2/repository/org/apache/maven/shared/maven-shared-utils/3.3.4/maven-shared-utils-3.3.4.jar:/home/ise/.m2/repository/commons-io/commons-io/2.11.0/commons-io-2.11.0.jar:/home/ise/.m2/repository/com/google/inject/guice/4.2.2/guice-4.2.2-no_aop.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-classworlds/2.6.0/plexus-classworlds-2.6.0.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-sec-dispatcher/2.0/plexus-sec-dispatcher-2.0.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-cipher/2.0/plexus-cipher-2.0.jar:/home/ise/.m2/repository/commons-cli/commons-cli/1.4/commons-cli-1.4.jar:/home/ise/.m2/repository/org/eclipse/sisu/org.eclipse.sisu.plexus/0.3.5/org.eclipse.sisu.plexus-0.3.5.jar:/home/ise/.m2/repository/org/apache/maven/maven-settings-builder/3.8.6/maven-settings-builder-3.8.6.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-interpolation/1.26/plexus-interpolation-1.26.jar:/home/ise/.m2/repository/org/apache/maven/maven-resolver-provider/3.8.6/maven-resolver-provider-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/maven-repository-metadata/3.8.6/maven-repository-metadata-3.8.6.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-spi/1.6.3/maven-resolver-spi-1.6.3.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-impl/1.6.3/maven-resolver-impl-1.6.3.jar:/home/ise/.m2/repository/org/codehaus/plexus/plexus-utils/3.3.0/plexus-utils-3.3.0.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-connector-basic/1.6.3/maven-resolver-connector-basic-1.6.3.jar:/home/ise/.m2/repository/org/apache/maven/resolver/maven-resolver-transport-wagon/1.6.3/maven-resolver-transport-wagon-1.6.3.jar:/home/ise/.m2/repository/org/apache/maven/wagon/wagon-http/3.5.1/wagon-http-3.5.1.jar:/home/ise/.m2/repository/org/apache/maven/wagon/wagon-http-shared/3.5.1/wagon-http-shared-3.5.1.jar:/home/ise/.m2/repository/org/apache/maven/wagon/wagon-provider-api/3.5.1/wagon-provider-api-3.5.1.jar:/home/ise/.m2/repository/org/apache/maven/wagon/wagon-file/3.5.1/wagon-file-3.5.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-bootstrap-gradle-resolver/2.15.0.Final/quarkus-bootstrap-gradle-resolver-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/jandex/3.0.5/jandex-3.0.5.jar:/home/ise/.m2/repository/io/quarkus/quarkus-junit5-properties/2.15.0.Final/quarkus-junit5-properties-2.15.0.Final.jar:/home/ise/.m2/repository/org/junit/jupiter/junit-jupiter/5.9.1/junit-jupiter-5.9.1.jar:/home/ise/.m2/repository/org/junit/jupiter/junit-jupiter-params/5.9.1/junit-jupiter-params-5.9.1.jar:/home/ise/.m2/repository/org/junit/jupiter/junit-jupiter-engine/5.9.1/junit-jupiter-engine-5.9.1.jar:/home/ise/.m2/repository/org/junit/platform/junit-platform-engine/1.9.1/junit-platform-engine-1.9.1.jar:/home/ise/.m2/repository/com/thoughtworks/xstream/xstream/1.4.19/xstream-1.4.19.jar:/home/ise/.m2/repository/io/github/x-stream/mxparser/1.2.2/mxparser-1.2.2.jar:/home/ise/.m2/repository/xmlpull/xmlpull/1.1.3.1/xmlpull-1.1.3.1.jar:/home/ise/.m2/repository/io/quarkus/quarkus-junit5-mockito/2.15.0.Final/quarkus-junit5-mockito-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-junit5-mockito-config/2.15.0.Final/quarkus-junit5-mockito-config-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-arc-deployment/2.15.0.Final/quarkus-arc-deployment-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-smallrye-context-propagation-spi/2.15.0.Final/quarkus-smallrye-context-propagation-spi-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/quarkus-vertx-http-dev-console-spi/2.15.0.Final/quarkus-vertx-http-dev-console-spi-2.15.0.Final.jar:/home/ise/.m2/repository/io/quarkus/arc/arc-processor/2.15.0.Final/arc-processor-2.15.0.Final.jar:/home/ise/.m2/repository/org/mockito/mockito-core/4.9.0/mockito-core-4.9.0.jar:/home/ise/.m2/repository/net/bytebuddy/byte-buddy/1.12.18/byte-buddy-1.12.18.jar:/home/ise/.m2/repository/net/bytebuddy/byte-buddy-agent/1.12.16/byte-buddy-agent-1.12.16.jar:/home/ise/.m2/repository/org/objenesis/objenesis/3.3/objenesis-3.3.jar:/home/ise/.m2/repository/io/quarkus/quarkus-mutiny/2.15.0.Final/quarkus-mutiny-2.15.0.Final.jar:/home/ise/.m2/repository/io/smallrye/reactive/mutiny-smallrye-context-propagation/1.8.0/mutiny-smallrye-context-propagation-1.8.0.jar:/home/ise/.m2/repository/io/rest-assured/rest-assured/4.5.1/rest-assured-4.5.1.jar:/home/ise/.m2/repository/org/codehaus/groovy/groovy/3.0.9/groovy-3.0.9.jar:/home/ise/.m2/repository/org/codehaus/groovy/groovy-xml/3.0.9/groovy-xml-3.0.9.jar:/home/ise/.m2/repository/org/apache/httpcomponents/httpclient/4.5.13/httpclient-4.5.13.jar:/home/ise/.m2/repository/org/apache/httpcomponents/httpcore/4.4.15/httpcore-4.4.15.jar:/home/ise/.m2/repository/org/apache/httpcomponents/httpmime/4.5.13/httpmime-4.5.13.jar:/home/ise/.m2/repository/org/hamcrest/hamcrest/2.1/hamcrest-2.1.jar:/home/ise/.m2/repository/org/ccil/cowan/tagsoup/tagsoup/1.2.1/tagsoup-1.2.1.jar:/home/ise/.m2/repository/io/rest-assured/json-path/4.5.1/json-path-4.5.1.jar:/home/ise/.m2/repository/org/codehaus/groovy/groovy-json/3.0.9/groovy-json-3.0.9.jar:/home/ise/.m2/repository/io/rest-assured/rest-assured-common/4.5.1/rest-assured-common-4.5.1.jar:/home/ise/.m2/repository/io/rest-assured/xml-path/4.5.1/xml-path-4.5.1.jar:/home/ise/.m2/repository/org/apache/commons/commons-lang3/3.12.0/commons-lang3-3.12.0.jar:/home/ise/.m2/repository/org/mockito/mockito-junit-jupiter/4.9.0/mockito-junit-jupiter-4.9.0.jar:/home/ise/.m2/repository/org/assertj/assertj-core/3.22.0/assertj-core-3.22.0.jar:/home/ise/.m2/repository/com/jayway/jsonpath/json-path/2.5.0/json-path-2.5.0.jar:/home/ise/.m2/repository/net/minidev/json-smart/2.3/json-smart-2.3.jar:/home/ise/.m2/repository/net/minidev/accessors-smart/1.2/accessors-smart-1.2.jar:/home/ise/.m2/repository/com/apollographql/apollo/apollo-runtime/2.2.3/apollo-runtime-2.2.3.jar:/home/ise/.m2/repository/com/apollographql/apollo/apollo-api-jvm/2.2.3/apollo-api-jvm-2.2.3.jar:/home/ise/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-common/1.7.21/kotlin-stdlib-common-1.7.21.jar:/home/ise/.m2/repository/com/apollographql/apollo/apollo-normalized-cache-jvm/2.2.3/apollo-normalized-cache-jvm-2.2.3.jar:/home/ise/.m2/repository/com/apollographql/apollo/apollo-normalized-cache-api-jvm/2.2.3/apollo-normalized-cache-api-jvm-2.2.3.jar:/home/ise/.m2/repository/com/benasher44/uuid-jvm/0.1.0/uuid-jvm-0.1.0.jar:/home/ise/.m2/repository/com/nytimes/android/cache/2.0.2/cache-2.0.2.jar:/home/ise/.m2/repository/com/apollographql/apollo/apollo-http-cache-api/2.2.3/apollo-http-cache-api-2.2.3.jar:/home/ise/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib/1.7.21/kotlin-stdlib-1.7.21.jar:/home/ise/.m2/repository/org/jetbrains/annotations/17.0.0/annotations-17.0.0.jar:/home/ise/.m2/repository/com/squareup/okhttp3/okhttp/4.10.0/okhttp-4.10.0.jar:/home/ise/.m2/repository/com/squareup/okio/okio-jvm/3.0.0/okio-jvm-3.0.0.jar:/home/ise/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-jdk8/1.7.21/kotlin-stdlib-jdk8-1.7.21.jar:/home/ise/.m2/repository/org/jetbrains/kotlin/kotlin-stdlib-jdk7/1.7.21/kotlin-stdlib-jdk7-1.7.21.jar:/home/ise/.m2/repository/com/datastax/oss/java-driver-core/4.14.1/java-driver-core-4.14.1.jar:/home/ise/.m2/repository/com/datastax/oss/native-protocol/1.5.1/native-protocol-1.5.1.jar:/home/ise/.m2/repository/io/netty/netty-handler/4.1.85.Final/netty-handler-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-common/4.1.85.Final/netty-common-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-resolver/4.1.85.Final/netty-resolver-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-buffer/4.1.85.Final/netty-buffer-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-transport/4.1.85.Final/netty-transport-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-transport-native-unix-common/4.1.85.Final/netty-transport-native-unix-common-4.1.85.Final.jar:/home/ise/.m2/repository/io/netty/netty-codec/4.1.85.Final/netty-codec-4.1.85.Final.jar:/home/ise/.m2/repository/com/datastax/oss/java-driver-shaded-guava/25.1-jre-graal-sub-1/java-driver-shaded-guava-25.1-jre-graal-sub-1.jar:/home/ise/.m2/repository/com/typesafe/config/1.4.1/config-1.4.1.jar:/home/ise/.m2/repository/com/github/jnr/jnr-posix/3.1.15/jnr-posix-3.1.15.jar:/home/ise/.m2/repository/com/github/jnr/jnr-ffi/2.2.11/jnr-ffi-2.2.11.jar:/home/ise/.m2/repository/com/github/jnr/jffi/1.3.9/jffi-1.3.9.jar:/home/ise/.m2/repository/com/github/jnr/jffi/1.3.9/jffi-1.3.9-native.jar:/home/ise/.m2/repository/org/ow2/asm/asm-analysis/9.4/asm-analysis-9.4.jar:/home/ise/.m2/repository/org/ow2/asm/asm-tree/9.4/asm-tree-9.4.jar:/home/ise/.m2/repository/org/ow2/asm/asm-util/9.4/asm-util-9.4.jar:/home/ise/.m2/repository/com/github/jnr/jnr-a64asm/1.0.0/jnr-a64asm-1.0.0.jar:/home/ise/.m2/repository/com/github/jnr/jnr-x86asm/1.0.2/jnr-x86asm-1.0.2.jar:/home/ise/.m2/repository/com/github/jnr/jnr-constants/0.10.3/jnr-constants-0.10.3.jar:/home/ise/.m2/repository/io/dropwizard/metrics/metrics-core/4.1.18/metrics-core-4.1.18.jar:/home/ise/.m2/repository/org/hdrhistogram/HdrHistogram/2.1.12/HdrHistogram-2.1.12.jar:/home/ise/.m2/repository/com/fasterxml/jackson/core/jackson-core/2.14.1/jackson-core-2.14.1.jar:/home/ise/.m2/repository/com/fasterxml/jackson/core/jackson-databind/2.14.1/jackson-databind-2.14.1.jar:/home/ise/.m2/repository/com/fasterxml/jackson/core/jackson-annotations/2.14.1/jackson-annotations-2.14.1.jar:/home/ise/.m2/repository/com/github/stephenc/jcip/jcip-annotations/1.0-1/jcip-annotations-1.0-1.jar:/home/ise/.m2/repository/com/github/spotbugs/spotbugs-annotations/3.1.12/spotbugs-annotations-3.1.12.jar:/home/ise/.m2/repository/com/google/code/findbugs/jsr305/3.0.2/jsr305-3.0.2.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit5 io.stargate.sgv2.graphql.schema.graphqlfirst.processor.QueryMappingModelTest
ANTLR Tool version 4.9.3 used for code generation does not match the current runtime version 4.10.1
ANTLR Runtime version 4.9.3 used for parser compilation does not match the current runtime version 4.10.1

java.lang.ExceptionInInitializerError
	at graphql.parser.Parser.parseImpl(Parser.java:195)
	at graphql.parser.Parser.parseDocumentImpl(Parser.java:164)
	at graphql.parser.Parser.parseDocument(Parser.java:155)
	at graphql.schema.idl.SchemaParser.parseImpl(SchemaParser.java:120)
	at graphql.schema.idl.SchemaParser.parseImpl(SchemaParser.java:111)
	at graphql.schema.idl.SchemaParser.parse(SchemaParser.java:106)
	at io.stargate.sgv2.graphql.schema.graphqlfirst.processor.QueryMappingModelTest.shouldBuildAMappingModelAndQueryWithOnePrimaryKey(QueryMappingModelTest.java:43)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.UnsupportedOperationException: java.io.InvalidClassException: org.antlr.v4.runtime.atn.ATN; Could not deserialize ATN with version 3 (expected 4).
	at org.antlr.v4.runtime.atn.ATNDeserializer.deserialize(ATNDeserializer.java:56)
	at org.antlr.v4.runtime.atn.ATNDeserializer.deserialize(ATNDeserializer.java:48)
	at graphql.parser.antlr.GraphqlLexer.<clinit>(GraphqlLexer.java:343)
	... 77 more
Caused by: java.io.InvalidClassException: org.antlr.v4.runtime.atn.ATN; Could not deserialize ATN with version 3 (expected 4).
	... 80 more


java.lang.NoClassDefFoundError: Could not initialize class graphql.parser.antlr.GraphqlLexer

	at graphql.parser.Parser.parseImpl(Parser.java:195)
	at graphql.parser.Parser.parseDocumentImpl(Parser.java:164)
	at graphql.parser.Parser.parseDocument(Parser.java:155)
	at graphql.schema.idl.SchemaParser.parseImpl(SchemaParser.java:120)
	at graphql.schema.idl.SchemaParser.parseImpl(SchemaParser.java:111)
	at graphql.schema.idl.SchemaParser.parse(SchemaParser.java:106)
	at io.stargate.sgv2.graphql.schema.graphqlfirst.processor.QueryMappingModelTest.shouldThrowWhenCreatingAQueryForUnknownEntity(QueryMappingModelTest.java:76)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.junit.platform.launcher.core.SessionPerRequestLauncher.execute(SessionPerRequestLauncher.java:53)
	at com.intellij.junit5.JUnit5IdeaTestRunner.startRunnerWithArgs(JUnit5IdeaTestRunner.java:57)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
```

--- a/apis/pom.xml
+++ b/apis/pom.xml
@@ -23,7 +23,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>2.14.3.Final</quarkus.platform.version>
+    <quarkus.platform.version>2.15.0.Final</quarkus.platform.version>
     <skipTests>false</skipTests>
     <skipUnitTests>${skipTests}</skipUnitTests>
     <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>

--- a/apis/pom.xml
+++ b/apis/pom.xml
@@ -23,7 +23,7 @@
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
     <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
-    <quarkus.platform.version>2.15.0.Final</quarkus.platform.version>
+    <quarkus.platform.version>2.15.1.Final</quarkus.platform.version>
     <skipTests>false</skipTests>
     <skipUnitTests>${skipTests}</skipUnitTests>
     <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>

--- a/apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/CollectionsResourceIntegrationTest.java
+++ b/apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/CollectionsResourceIntegrationTest.java
@@ -23,7 +23,6 @@ import static org.hamcrest.Matchers.empty;
 import static org.hamcrest.Matchers.endsWith;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.nullValue;
-import static org.hamcrest.Matchers.startsWith;
 
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusIntegrationTest;
@@ -166,7 +165,7 @@ public class CollectionsResourceIntegrationTest extends DocsApiIntegrationTest {
           .then()
           .statusCode(400)
           .body("code", is(400))
-          .body("description", startsWith("Unable to process JSON"));
+          .body("description", is("HTTP 400 Bad Request"));
     }
 
     @Test

--- a/apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentUpdateResourceIntegrationTest.java
+++ b/apis/sgv2-docsapi/src/test/java/io/stargate/sgv2/docsapi/api/v2/namespaces/collections/documents/DocumentUpdateResourceIntegrationTest.java
@@ -23,7 +23,6 @@ import static net.javacrumbs.jsonunit.JsonMatchers.jsonPartEquals;
 import static net.javacrumbs.jsonunit.core.util.ResourceUtils.resource;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.notNullValue;
-import static org.hamcrest.Matchers.startsWith;
 import static org.testcontainers.shaded.org.awaitility.Awaitility.await;
 
 import com.google.common.io.CharStreams;
@@ -391,7 +390,7 @@ class DocumentUpdateResourceIntegrationTest extends DocsApiIntegrationTest {
           .then()
           .statusCode(400)
           .body("code", is(400))
-          .body("description", startsWith("Unable to process JSON"));
+          .body("description", is("HTTP 400 Bad Request"));
     }
 
     @Test

--- a/apis/sgv2-graphqlapi/pom.xml
+++ b/apis/sgv2-graphqlapi/pom.xml
@@ -36,6 +36,13 @@
       <artifactId>graphql-java</artifactId>
       <version>18.3</version>
     </dependency>
+    <!-- Quarkus sets it's own antlr version, keep this is sync with graphql-java deps -->
+    <dependency>
+      <groupId>org.antlr</groupId>
+      <artifactId>antlr4-runtime</artifactId>
+      <version>4.9.3</version>
+      <scope>runtime</scope>
+    </dependency>
     <dependency>
       <groupId>com.apollographql.federation</groupId>
       <artifactId>federation-graphql-java-support</artifactId>

--- a/pom.xml
+++ b/pom.xml
@@ -823,7 +823,6 @@
         <artifactId>nimbus-jose-jwt</artifactId>
         <version>9.21</version>
       </dependency>
-
     </dependencies>
   </dependencyManagement>
  

--- a/pom.xml
+++ b/pom.xml
@@ -39,7 +39,7 @@
 
     <caffeine.version>2.9.3</caffeine.version>
     <commons-io.version>2.7</commons-io.version>
-    <grpc.version>1.50.2</grpc.version>
+    <grpc.version>1.51.0</grpc.version>
     <immutables.version>2.8.8</immutables.version>
     <jackson.version>2.13.4</jackson.version>
     <javatuples.version>1.2</javatuples.version>

